images/metering-ansible-operator/roles/meteringconfig/tasks/reconcile.yml

```diff
@@ -7,10 +7,10 @@
 
 - name: Validate Hive storage configuration
   fail:
```

Comment:
we should update this to use the `assert` module, which is already extensively used throughout this role:
```
- name: Validate Hive storage configuration
  assert:
    that:
      - hiveStorageType is defined
      - hiveStorageType in ['s3', 'sharedPVC', 'hdfs', 'azure']
    msg: "Invalid spec.storage.hive.type: '{{ hiveStorageType }}', must be one of 's3', 'azure', or 'sharedPVC'"
```

Reply:
This isn't a big deal, just something that I noticed, and can be changed in future PRs.

Reply:
We can do this in a separate PR